src/StatusQ/Controls/StatusInput.qml

```diff
@@ -347,9 +352,7 @@ Item {
     }
 
     implicitWidth: 448
-    implicitHeight: ((internal.inputHeight + topRow.height + errorMessage.height) +
-                    ((topRow.height > 0) && (errorMessage.height > 0) ? 16 :
-                    (topRow.height > 0) || (errorMessage.height > 0) ? 8 : 0))
+    implicitHeight: inputLayout.implicitHeight
 
```

Comment:
could you please let me know how this works, `StatusInput` takes it's height from `ColumnLayout` but  `ColumnLayout` is set to `anchors.fill: parent`?

Reply:
StatusInput takes `implicit` size from ColumnLayout what is calculated from implicit sizes of layout's children, spacing and margins. So there is no need to calculate it manually. `anchors.fill: parent` affects explicit size (width and height properties), not implicit one. [Here](https://stackoverflow.com/questions/45946808/difference-between-width-height-and-implicitwidth-height-and-corresponding-use) is good answer.

Reply:
Are you sure a Column/Layout has an implicitHeight?

Reply:
it looks and feels weird. How about something like:
```
        width: parent.width
        height: childrenRect.height
```
instead of `anchors.fill: parent`?

Reply:
@caybro yes, otherwise it was not working :) @alexandraB99 `height: childrenRect.height` can cause a binding loop. 
I'm strongly believe that using implicit sizes of reusable components is a good practice because we always have two flows and mixing them together won't work properly in every places. And this is actually the reason that we getting problems with layouts. Most modern QML libraries like QQC2 and QQL using implicit sizing flow. And it's mentioned in the [documentation](https://doc.qt.io/qt-6/qtquicklayouts-overview.html).

Reply:
Ok, I'll trust you on that :) Otherwise another :beer: :D

Reply:
@caybro can you please approve then?)